packages/editor-ui/src/components/ResourceLocator/ResourceLocator.vue

```diff
@@ -429,6 +433,18 @@ export default mixins(debounceHelper, workflowHelpers, nodeHelpers).extend({
 				this.$emit('input', { ...this.value, __regex: mode.extractValue.regex });
 			}
 		},
+		dependentParametersValues() {
```

Comment:
where are the tests? 
![7VE](https://user-images.githubusercontent.com/4711238/234868221-9dd5dca8-9248-4dbd-833d-9ded42f630e9.gif)